packages/legend-graph/src/graph-manager/protocol/pure/v1/transformation/pureGraph/to/V1_ElementFirstPassBuilder.ts

```diff
@@ -159,6 +160,9 @@ export class V1_ElementFirstPassBuilder
     metamodel.applicationName = element.applicationName;
     metamodel.description = element.description;
     metamodel.owner = element.owner;
+    if (element.type) {
+      metamodel.type = element.type.valueOf() as SnowflakeAppType;
```

Comment:
is this how we handle enums on the rest of the builders?
Don't really like casting here. how do we validate that the type if an accepted enum value.